app/views/event_groups/_fields.html.erb

```diff
@@ -30,6 +30,20 @@
 	<%= f.input :mentor_confirmation_message, :input_html => { :rows => 6 }, :label => "#{Customer.mentor_Label} confirmation message" %>
 <% end -%>
 
+<%= f.inputs "RSVP Cancellation Disabling" do %>
+	<p>Setting one of the below will disable members of the specified audience from cancelling their RSVPs for events in this group.
+	<br />
+	<br />
+	Note that a message must be set for that audience if a number of hours is set. This message will be displayed in place of a cancel button, e.g. "Please contact Elmo at Sesame Street to cancel your RSVP."</p>
+
+	<%= f.input :mentor_hours_prior_disable_cancel, :label => "Hours before to disable for #{Customer.mentors_label}" %>
```

Comment:
You could use the inline input fields here (`f.inputs :name => 'Disable time', :class => 'inline'`) but this is fine.